variables.ini.dist

```diff
@@ -1,147 +1,146 @@
+REPLICATION_PROVIDER_LOCATION = # iPlant Cloud - Tucson
+MONTHLY_RESET_PROVIDER_LOCATIONS = [] # ['iPlant Cloud - Tucson']
+USER_EMAIL_LOOKUP_METHOD = # ldap_get_email_info
+EMAIL_LOOKUP_METHOD = # ldapLookupEmail
+DEFAULT_EMAIL_DOMAIN = # iplantcollaborative.org
+DEFAULT_ALLOCATION_THRESHOLD = (7 * 24 * 60)  # In Minutes
+DEFAULT_ALLOCATION_DELTA = (365 * 24 * 60)  # In Minutes
+DEFAULT_QUOTA = {} # {'cpu': 16, 'memory': 128, 'storage': 10}
+DEFAULT_KEYSTONE_ROLE = # _member_
+DEFAULT_IP_LOOKUP = # 'iplant_hostname'
+DEFAULT_PASSWORD_UPDATE = # 'keystone_password_update', 'openstack_sdk_password_update'
+INSTANCE_HOSTNAMING_DOMAIN = # iplantcollaborative.org
+INSTANCE_HOSTNAMING_FORMAT = # vm%(three)s-%(four)s.%(domain)s
+DEPLOYMENT_KEYPAIR_NAME = # keypair name inject by atmo-ansible during instance deploy
+EXPIRATION_PLUGINS = ["atmosphere.plugins.auth.validation.NeverExpire"] # list of plugins
+VALIDATION_PLUGINS = ["atmosphere.plugins.auth.validation.AlwaysAllow"] # list of plugins
+SENTRY_ENABLED =  True # Boolean required or False
+SENTRY_DSN = # https://...@sentry.io/....
 
 #NOTE: Jetstream Specific
-TACC_API_USER = ; "tas-username"
-TACC_API_PASS = ; "tas-password"
-TACC_API_URL  = ; "/path to api/"
+TACC_API_USER = # "tas-username"
+TACC_API_PASS = # "tas-password"
+TACC_API_URL  = # "/path to api/"
 
 #NOTE: Their are *REQUIRED* quotations around 'timedelta(..)'!
-CELERYBEAT_SCHEDULE = ; {"monitor_instances": {"schedule": 'timedelta(minutes=5)',}}
+CELERYBEAT_SCHEDULE = {} # {"monitor_instances": {"schedule": 'timedelta(minutes=5)',}}
 
 [secrets.py]
-IRODS_HOST = ; irods.server.com
-IRODS_PORT = ; 8000
-IRODS_ZONE = ; irods_zone
-IRODS_ADMIN_USER = ; irods_user
-IRODS_ADMIN_PASS = ; irods_password
-LOCAL_STORAGE = ; /storage
-CHROMO_DEPLOYMENT_KEY = ; ssh-rsa AAA... example@deploykey.com
-CHROMO_EXTRA_IMAGE_METADATA = ; Example {"key1": "value1", "key2": "yes"}
-ATMOSPHERE_SUPERUSER = ; atmo_superuser
-ATMOSPHERE_SECRET_SEED = ; (Random value:) 9oe5q7cp714zmmjl51
-ATMOSPHERE_VNC_LICENSE = ; vnc_license_key
+IRODS_HOST = # irods.server.com
+IRODS_PORT = # 8000
+IRODS_ZONE = # irods_zone
+IRODS_ADMIN_USER = # irods_user
+IRODS_ADMIN_PASS = # irods_password
+LOCAL_STORAGE = # /storage
+CHROMO_DEPLOYMENT_KEY = # ssh-rsa AAA... example@deploykey.com
+CHROMO_EXTRA_IMAGE_METADATA = {} # {"key1": "value1", "key2": "yes"}
+ATMOSPHERE_SUPERUSER = # atmo_superuser
+ATMOSPHERE_SECRET_SEED = # (Random value:) 9oe5q7cp714zmmjl51
+ATMOSPHERE_VNC_LICENSE = # vnc_license_key
 
 [celeryd.default]
-ATMOSPHERE_LOCATION= ; /path/to/atmosphere
-SSL_LOCATION= ; /etc/ssl
-SSL_CERTIFICATE= ; selfsigned_server.crt
-SSL_KEY= ; selfsigned_server.key
-USE_PRODUCTION= ; True
-FLOWER_AUTH_SCHEME=; "basic" or "google"
+ATMOSPHERE_LOCATION= # /path/to/atmosphere
+CELERYD_TLS_PRIVKEY_DEST = # "/etc/ssl/private/name_of_your_org.key"
+CELERYD_TLS_CERT_DEST = # "/etc/ssl/certs/name_of_your_org.crt"
+USE_PRODUCTION=  True # Boolean required
+VIRTUALENV_PATH = # ""
```

Comment:
Could there be a better example than `""`, that doesn't give much insight about how to format what goes here